extensions/sql-bindings/src/common/constants.ts

```diff
@@ -69,11 +69,11 @@ export const connectionProfile = localize('connectionProfile', 'Select a connect
 export const userConnectionString = localize('userConnectionString', 'Enter connection string');
 export const selectConnectionString = localize('selectConnectionString', 'Select SQL connection string method');
 export const includePassword = localize('includePassword', 'Do you want to include the password from this connection in your local.settings.json file?');
-export const enterPasswordPrompt = localize('enterPasswordPrompt', 'Enter the password to be used for the connection string');
-export const enterPasswordManually = localize('enterPasswordManually', 'Enter password or press escape to cancel');
 export const userPasswordLater = localize('userPasswordLater', 'In order to user the SQL connection string later you will need to manually enter the password in your local.settings.json file.');
 export const openFile = localize('openFile', "Open File");
 export const closeButton = localize('closeButton', "Close");
+export const enterPasswordPrompt = localize('enterPasswordPrompt', 'Enter the password to be used for the connection string that will be saved in the local.settings.json file');
+export const enterPasswordManually = localize('enterPasswordManually', 'Enter password or press escape to cancel');
```

Comment:
Doesn't the input prompt already tell the user they can pretty escape to cancel?

Reply:
![image](https://user-images.githubusercontent.com/23587151/168989985-40b67cc1-2aee-4925-9a8b-80d0c3440757.png)

Reply:
Yes, exactly. It's already included in the text (`Press 'Enter' to confirm or 'Escape' to cancel`) so it's just unnecessary duplication to have it in the placeholder text as well. I'd just remove that whole placeholder, it's not adding anything useful.